include/zephyr/drivers/charger/charger_sbs_charger.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DRIVERS_CHARGER_SBS_CHARGER_H_
+#define ZEPHYR_INCLUDE_DRIVERS_CHARGER_SBS_CHARGER_H_
+
+#include <zephyr/drivers/i2c.h>
+
+#define SBS_CHARGER_REG_SPEC_INFO     0x11
+#define SBS_CHARGER_REG_STATUS        0x13
+#define SBS_CHARGER_REG_ALARM_WARNING 0x16
+
+#define SBS_CHARGER_STATUS_CHARGE_INHIBITED BIT(0)
+#define SBS_CHARGER_STATUS_RES_COLD         BIT(9)
+#define SBS_CHARGER_STATUS_RES_HOT          BIT(10)
+#define SBS_CHARGER_STATUS_BATTERY_PRESENT  BIT(14)
+#define SBS_CHARGER_STATUS_AC_PRESENT       BIT(15)
+
+#define SBS_CHARGER_POLL_TIME 500
+
+struct sbs_charger_config {
+	struct i2c_dt_spec i2c;
+};
```

Comment:
This should not be in `include/zephyr/drivers`, that is for defines used by code external to the driver. Either bring it inside the driver code itself (it's single file, so why not) or in a header in the same directory.

Reply:
ack

Reply:
still relevant, this should be moved

Reply:
sorry I missed this one, I forgot to address it in the big rewrite that just happened. ack.

Reply:
actually, I forgot that those register defines are used by `emul_sbs_charger.c`. I can move `struct sbs_charger_config` to. `sbs_charger.c`, but do you mind if I leave the defines in `charger_sbs_charger.h`?

Reply:
this is used by `drivers/charger/sbs_charger.c` and `drivers/charger/emul_sbs_charger.c` right? Then just move the file in `drivers/charger/sbs_charger.h` and include it with `#include "charger_sbs_charger.h"`. Fine to leave the config struct there if you prefer though I would rather keep the scope as narrow as possible and declare it in the file itself if it's not needed in the emul code.

Reply:
ack. I will recreate a header file at `drivers/charger/sbs_charger.h` and move the defines in there.